docs/website/docs/user-guides/quickstart.md

```diff
@@ -149,7 +149,54 @@ $ go mod init my.example.go.project
 
 Your source code has now been generated and created in the directory.
 
-## Step 1. Creating your application (`odo init`)
+
+## Step 1. Connect to your cluster and create a new namespace or project"
+
+Before starting you should make sure that odo is connected to your cluster and that you have created a new namespace (or project if you are using OpenShift).
+
+
+
+
+<Tabs groupId="quickstart">
+  <TabItem value="kubernetes" label="Kubernetes">
+
+### Creating a new namespace
+
+If you are using Kubernetes, you can create a new namespace with the `odo create namespace` command.
+
+```console
+$ odo create namespace odo-dev
+ ✓  Namespace "odo-dev" is ready for use
+ ✓  New namespace created and now using namespace: odo-dev
+```
+
+  </TabItem>
+  <TabItem value="openshift" label="OpenShift">
+
+### Login to OpenShift Cluster
+
+The easies way to connect `odo` to an OpenShift cluster is use copy "Copy login command" function in OpenShift Web Console.
+
+1. Login to OpenShift Web Console.
+2. At the top right corner click on your username and than on "Copy login command".
+3. You will be prompted to enter your login credentials again.
+4. After login open "Display Token" link.
+5. Copy whole `oc login --token ...` command and paste it into the terminal, **before executing the command replace `oc` with `odo`.**
```

Comment:
>  \*\*before executing the command replace `oc` with `odo`.\*\*

This part is optional.

Reply:
Not really, as we can't be sure that the user has `oc` installed. But we can be sure that the user has `odo` installed at this point.